docs/tutorials/projects/tipsy.md

```diff
@@ -11,7 +11,7 @@ mp4Src: "/tutorials/tipsy/tipsy-preview.mp4"
 authors: ["Hazal Mestci"]
 languages: ["python"]
 viamresources:
-  ["board", "motor", "base", "camera", "sensor", "mlmodel", "vision"]
+  ["board", "motor", "base", "camera", "sensor", "mlmodel", "vision", "sensors"]
```

Comment:
are both the sensor and sensors necessary?

Reply:
yes - they are how the filtering works on https://docs.viam.com/tutorials/ . Sensor is the component, sensors the serivce

Reply:
makes sense!